### ISSUE TITLE
compatibility with async/await syntax

### ISSUE THREAD

**Issue by sylvainpolletvillard** [lukeed/taskr#8]
Hello,

Is it possible to use the ES7 async/await syntax with fly ? I have seen it is already available for node with https://github.com/yortus/asyncawait , without any major gotchas. I really like this syntax compared to yield/askterisks, and I think it would ease the transition when ES7 specifications will come to Node.js (presumably end 2016 ?). What do you think ?

**Comment by ghost** [comment]
@sylvainpolletvillard It could be possible and if you look in the examples, you will find that the `babel`'s build task is using the `stage: 0` flag that supports a bunch of ES7 features. I still need to come around with the docs, support for ES6 and ES7 Flyfiles out of the box (I mean you could do this right now, but it would be up to you transpiling the Flyfile) and specific ES7 examples, but we will be set pretty soon.

> What do you think ?

That's exactly the plan and thanks for opening this.

Very soon!

``` js
export async function build () {
  await this.source("...")...target("...")
}
```

**Comment by sylvainpolletvillard** [comment]
Nice ! I look forward to see more. Fly is the first build system I see with such a seamless API. I have not even started but it looks already natural to work with.

**Comment by sholladay** [comment]
+1, I'm excited for this. What's the best way to contribute for those of us who don't have time for a big PR? Would a bounty be useful?

**Comment by bucaran** [comment]
@sylvainpolletvillard @sholladay Support fort this landed in `0.1.0`

`0.1.1` will include more consisten node version support and other stuff.

**Comment by ghost** [comment]
@sholladay @sylvainpolletvillard Landed in 0.1.0.

Now it's OK to:

``` js
export async function build () {
  await this.source("...")...target("...")
}
```

0.1.1 will include more consistent node support.

@sholladay What about plugins or improving the Wiki? Even writing a blog article would be really helpful.

Thanks!
